source/command-monitoring/tests/find.json

```diff
@@ -0,0 +1,239 @@
+{
+  "data": [
+    {
+      "_id": 1,
+      "x": 11
+    },
+    {
+      "_id": 2,
+      "x": 22
+    },
+    {
+      "_id": 3,
+      "x": 33
+    },
+    {
+      "_id": 4,
+      "x": 44
+    },
+    {
+      "_id": 5,
+      "x": 55
+    }
+  ],
+  "tests": [
+    {
+      "description": "A successful find event with no options",
+      "operation": {
+        "name": "find",
+        "arguments": {
+          "filter": {
+            "_id": 1
+          }
+        }
+      },
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "find": "values",
+              "filter": {
+                "_id": 1
+              }
+            },
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
+            "reply": [
+              {
+                "_id": 1,
+                "x": 11
+              }
+            ],
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        }
+      ]
+    },
+    {
+      "description": "A successful find event with options",
+      "operation": {
+        "name": "find",
+        "arguments": {
+          "filter": {
+            "_id": 1
+          },
+          "sort": {
+            "_id": 1
+          },
+          "skip": 2,
+          "limit": 2
```

Comment:
Let's add batch size into the mix, since implementations will have to be able to unravel numberToReturn calculations on the OP_QUERY into limit/batchSize on the find command